docs/buildah-build.1.md

```diff
@@ -820,6 +833,16 @@ buildah bud --platform linux/arm64 --platform linux/amd64 --manifest myimage /tm
 
 buildah bud --all-platforms --manifest myimage /tmp/mysrc
 
+### Building an image using (--output) custom build output
+
+buildah build -o out .
```

Comment:
(Personal rant: Ugh, so the syntax of --output is a set of key-value pairs, _except when it’s not_? I hate ambiguities.)

Reply:
this is just to match the short-hand syntax which `docker buildx` supports.

Reply:
The integration test should probably check that we don't unintentionally start rejecting this variation.

Reply:
(Marked unresolved to add that test, or to explicitly decide that we don’t need one.)

Reply:
added a test.